### PR TITLE
Invalid csrf token fix

### PR DESCRIPTION
**Pull request by Noderino**
I forgot to create a branch before sending my previous PR. And I think this will also include previous commits. Sorry for the mistake.

### CODE REVIEW

**Comment by Noderino** [comment]
I found why my first commit did not solve the issue. If you set `ListenForEvents` to true before calling `Connect` you will get `Invalid Csrf token` error when you call `GetStatus`.
```javascript
[ {
"error": {
  "type": "4107", 
  "message": "Invalid Csrf token, csrf_hash='xxx'"
}, 
"version": 9, 
"client_version": "1.0.51.693.g6ea1e7f6", 
"running": true
} ]
```

File: SpotifyAPI.Example/LocalControl.cs [hunk 1]
```diff
@@ -70,8 +85,8 @@ public void UpdateInfos()
             versionLabel.Text = status.Version.ToString();
             repeatShuffleLabel.Text = status.Repeat + @" and " + status.Shuffle;
 
-            if (status.Track != null) //Update track infos
-                UpdateTrack(status.Track);
+            //if (status.Track != null) //Update track infos
+            //    UpdateTrack(status.Track);
```
Comment (JohnnyCrazy) [inline SpotifyAPI.Example/LocalControl.cs]: Any specific reason for this out-comment?

Comment (Noderino) [inline SpotifyAPI.Example/LocalControl.cs]: [SpotifyLocalAPI.cs#103](https://github.com/JohnnyCrazy/SpotifyAPI-NET/pull/140/files#diff-8aafce16036626e8c5113f4188ed4e2eR103) this line ignores old status response if it is raising event for the first time. So we can get track information through OnTrackChange event when application started. Those lines became unnecessary.

Comment (Noderino) [inline SpotifyAPI.Example/LocalControl.cs]: I also made some design changes to SpotifyLocalAPI.cs and added new events. I will upload them in a few days to discuss with you. But i need to test a few things to be sure.

**Pull request closed without merging**